recipes/pyscaffoldext-markdown/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "pyscaffoldext-markdown" %}
+{% set version = "0.4" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyscaffoldext-markdown-{{ version }}.tar.gz
+  sha256: e3b93316619808f00f04b15d83fc8e938a9ef3e7fab81803a087159b6fcdab57
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - setuptools_scm
+  run:
+    - importlib-metadata
+    - pyscaffold >=4.0b4,<5.0a0
+    - python >=3.6
+    - recommonmark
+    - wheel >=0.31
+
+test:
+  imports:
+    - pyscaffoldext
+    - pyscaffoldext.markdown
+  commands:
+    - git config --global user.email "tester@testing.com"
```

Comment:
I guess this is needed to run pytest? Can you add a comment about it (and maybe comment it until the mentioned issue is fixed)

Reply:
Yes, pytest runs git, so I need to configure it for the tests. I will add a comment about this.

The issue I mention below is unrelated to the git test, so I'm of the opinion that commenting out the `git config` would lead to more confusion.

Is there any potential conflict or security issue with running `git config` in the test environment? I've been assuming that the test environment should be reasonably isolated.

Reply:
Looking into it further, I don't think `git` should run. I'm going to have to look into it some more. :(

Reply:
Despite my latest push I am still investigating...

Reply:
> Is there any potential conflict or security issue with running git config in the test environment?

In the CI it is though people can still debug recipes locally with `conda build -m .ci_support/something.yaml recipe/` (especially useful on macOS/Windows builds where docker isn't an option). If it is really needed then fair enough though I'd be tempted to not bother running the unit tests at all.

The recipe `tests` should only aim to catch recipe issues, broken code is upstream's problem and will hopefully be caught by their own CI. It can vary but in general the combination of
* `imports`
* `pip check`
* `command --help` (if entrypoints are used)

is enough to catch packaging bugs in Python packages and the unit test failures in conda-forge normally represent buggy tests from assumptions about the environment they're ran in.

Reply:
Hi @chrisburr and @maresb, thank you very much for looking into this... (I am one of PyScaffold's maintainers)

I can confirm that PyScaffold runs `git` under the hood, even outside the test suite (and therefore PyScaffold extensions will also depend on it).

Reply:
Okay @chrisburr, after messing around for a while I gave up on doing any unit tests and simply followed your alternative suggestion. Okay to deploy now?